### PR TITLE
Convert status embed controller to request spec

### PR DESCRIPTION
**Pull request by mjankowski**
The statuses controller spec is quite larger, so going to do it in pieces.

This one converts the `embed` action specs.

### CODE REVIEW

File: spec/requests/statuses/embed_spec.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Status embed' do
+  describe 'GET /users/:account_username/statuses/:id/embed' do
+    subject { get embed_account_status_path(account.username, status) }
```
Comment (ClearlyClaire) [inline spec/requests/statuses/embed_spec.rb:7]: I think we may want to either change the description, or hardcode the path (I have a slight preference for the latter). As is, the two can get out of sync without that being noticed.

**Review by ClearlyClaire — DISMISSED** [review]
Looks ok, with a small caveat.

File: spec/requests/statuses/embed_spec.rb (hunk 1 above)
Comment (mjankowski) [reply spec/requests/statuses/embed_spec.rb]: Point taken on possible divergence... in an ideal world we'd have sufficient routing specs to be confident in using these in the request specs, but short of that - fine to use the full paths here. Updated to use that style.

**Review by ClearlyClaire — APPROVED** [review]

**Comment by mjankowski** [comment]
This one's friend - https://github.com/mastodon/mastodon/pull/32449 - has been rebased and is primed for action.

**Pull request merged**